test/passing/doc_comments.mli

```diff
@@ -62,24 +66,37 @@ val doc_to_plain :
   -> string
 (** [doc_to_plain b ~subst s] using [b], subsitutes in [s] variables by
     their [subst] definition and renders cmdliner directives to plain text.
+
     @raise Invalid_argument in case of illegal syntax. *)
 
 val k : k
 (** this is a comment
+
```

Comment:
Are these extra open lines intended / necessary?

Reply:
Not intended. I added the blank line to separate lists and paragraphs. I'll add a special case for tags (and module references, verbatim and code blocks).

Reply:
There are sill new empty lines

Reply:
ignore me. I didn't see the the reference file.